src/edutap/wallet_google/api.py

```diff
@@ -349,13 +349,27 @@ def listing(
         if response.status_code != 200:
             raise Exception(f"Error: {response.status_code} - {response.text}")
 
-        data = json.loads(response.content)
```

Comment:
Actually it would be good to have here a model and validate instead of custom validation handling.